<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>936</width>
    <height>609</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>qTaggle</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="FileList" name="fileList">
       <attribute name="title">
        <string>Files</string>
       </attribute>
      </widget>
      <widget class="TagList" name="tagList">
       <attribute name="title">
        <string>Tags</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>936</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuOpenRecent">
     <property name="title">
      <string>Open recent</string>
     </property>
    </widget>
    <addaction name="actionAddFile"/>
    <addaction name="actionCreateTag"/>
    <addaction name="separator"/>
    <addaction name="actionNewDatabase"/>
    <addaction name="actionOpenDatabase"/>
    <addaction name="menuOpenRecent"/>
    <addaction name="actionCloseDatabase"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionEditSelected"/>
    <addaction name="actionCheckSelected"/>
    <addaction name="actionDeleteSelected"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionOptions"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAboutQt"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAddFile"/>
   <addaction name="actionCreateTag"/>
   <addaction name="separator"/>
   <addaction name="actionEditSelected"/>
   <addaction name="actionCheckSelected"/>
   <addaction name="actionDeleteSelected"/>
   <addaction name="separator"/>
   <addaction name="actionNewDatabase"/>
   <addaction name="actionOpenDatabase"/>
   <addaction name="actionCloseDatabase"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget">
   <property name="windowTitle">
    <string>Filters</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents"/>
  </widget>
  <widget class="QDockWidget" name="filePreviewDock">
   <property name="windowTitle">
    <string>Preview</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2"/>
  </widget>
  <widget class="QDockWidget" name="propertiesDock">
   <property name="windowTitle">
    <string>Properties</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_1"/>
  </widget>
  <action name="actionAddFile">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::DocumentOpen"/>
   </property>
   <property name="text">
    <string>Add file...</string>
   </property>
   <property name="toolTip">
    <string>Add file</string>
   </property>
  </action>
  <action name="actionCreateTag">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::ListAdd"/>
   </property>
   <property name="text">
    <string>Add tag...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::SystemShutdown"/>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="toolTip">
    <string>Exit qTaggle</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::QuitRole</enum>
   </property>
  </action>
  <action name="actionEditSelected">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::MailMessageNew"/>
   </property>
   <property name="text">
    <string>Edit selected...</string>
   </property>
  </action>
  <action name="actionNewDatabase">
   <property name="text">
    <string>New database...</string>
   </property>
  </action>
  <action name="actionOpenDatabase">
   <property name="text">
    <string>Open database...</string>
   </property>
  </action>
  <action name="actionCloseDatabase">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Close database</string>
   </property>
  </action>
  <action name="actionDeleteSelected">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::EditDelete"/>
   </property>
   <property name="text">
    <string>Delete selected</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>About Qt</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::AboutQtRole</enum>
   </property>
  </action>
  <action name="actionCheckSelected">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="view-refresh"/>
   </property>
   <property name="text">
    <string>Check file</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionOptions">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::DocumentProperties"/>
   </property>
   <property name="text">
    <string>Options...</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::PreferencesRole</enum>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TagList</class>
   <extends>QWidget</extends>
   <header>app/gui/taglist.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FileList</class>
   <extends>QWidget</extends>
   <header>app/gui/filelist.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
